app/uk/gov/hmrc/serviceconfigs/model/DeploymentConfigSnapshot.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.serviceconfigs.model
+
+import java.time.LocalDate
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, __}
+
+final case class DeploymentConfigSnapshot(
+  name: String,
+  date: LocalDate,
```

Comment:
Thinking about it, there's no need to fix the granularity to daily at this level, so happy to change to `LocalDateTime` if that makes more sense.

Reply:
yeah, no need to put the chosen frequency in the model. Maybe `Instant` would be simpler.

Reply:
It seems `LocalDateTime` is used elsewhere in the codebase and `Instant` is not, so I went for that.